search/routes/api/serialize.py

```diff
@@ -118,35 +124,92 @@ def as_json(document_or_set: Union[DocumentSet, Document],
 
 
 
-# TODO: implement me!
 class AtomXMLSerializer(BaseSerializer):
     """Atom XML serializer for paper metadata."""
 
-    ATOM = "http://www.w3.org/2005/Atom"
-    OPENSEARCH = "http://a9.com/-/spec/opensearch/1.1/"
-    ARXIV = "http://arxiv.org/schemas/atom"
-    NSMAP = {
-        None: ATOM,
-        "opensearch": OPENSEARCH,
-        "arxiv": ARXIV
-    }
-#     fields = {
-#         'title': '{%s}title' % ATOM,
-#         'id': '{%s}id' % ATOM,
-#         'submitted_date': '{%s}published' % ATOM,
-#         'modified_date': '{%s}updated' % ATOM,
-#         'abstract': '{%s}summary' % ATOM,
-#         ''
-#     }
-#
-#     def __init__(cls, *args, **kwargs) -> None:
-#         super(AtomXMLSerializer, cls).__init__(*args, **kwargs)
-#         cls._root = etree.Element('feed', nsmap=cls.NSMAP)
-#
-#     def transform(cls):
-#         for document in cls.iter_documents():
-#
-#
-#
-#     def __repr__(cls) -> str:
-#         return etree.tostring(cls._root, pretty_print=True)
+    @classmethod
+    def transform_document(cls, fg: FeedGenerator, doc: Document,
+                           query: Optional[APIQuery] = None) -> None:
+        """Select a subset of :class:`Document` properties for public API."""
+        entry = fg.add_entry()
+        entry.id(url_for("abs", paper_id=doc['paper_id'],
+                             version=doc['version'], _external=True))
+        entry.title(doc['title'])
+        entry.summary(doc['abstract'])
+        entry.published(doc['submitted_date'])
+        entry.updated(doc['updated_date'])
+        entry.link({'href': url_for("abs", paper_id=doc['paper_id'], 
+                                    version=doc['version'], _external=True),
+                    "type": "text/html"})
+        
+
+        entry.link({'href': url_for("pdf", paper_id=doc['paper_id'], 
+                                    version=doc['version'], _external=True),
+                    "type": "application/pdf", 'rel': 'related'})
+
+        if doc['comments']:
+            entry.arxiv.comment(doc['comments'])
+
+        if doc['journal_ref']:
+            entry.arxiv.journal_ref(doc['journal_ref'])
+
+        if doc['doi']:
+            entry.arxiv.doi(doc['doi'])
+
+        entry.arxiv.primary_category(doc['primary_classification']['archive']['id'])
+        entry.category(
+            term=doc['primary_classification']['archive']['id'],
+            scheme=ARXIV_NS)
```

Comment:
```suggestion
            scheme=ARXIV_NS
        )
```

Continuation lines should be either:

```python
foo = bar(baz,
          bat)
```
or
```python
foo = bar(
    baz,
    bat
)
```